README.md

```diff
@@ -225,7 +225,7 @@ Try `rpg --help` for more options and check the [shell integration guide](shell/
 
 ## Customize character classes
 
-The character class determines a character's initial stats and at what pace they increase when leveling up. By default, rpg-cli will use classes as defined by [this file](src/character/classes.yaml), but these definitions can be overridden by placing a YAML file with that same structure at `~/.rpg/classes.yaml`.
+The character class determines a character's initial stats and at what pace they increase when leveling up. By default, rpg-cli will use classes as defined by [this file](src/character/classes.yaml), but these definitions can be overridden by placing a YAML file with that same structure at `~/.local/share/rpg/classes.yaml`.
```

Comment:
It's ok to leave the `~/.local/share/` one as the example, but a reference to [this doc](https://docs.rs/dirs/3.0.2/dirs/fn.data_dir.html) would also be good, considering that can change depending on the OS and the env.